ci/cleanup_images.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -euo pipefail
+
+org="rapidsai"
+repo="staging"
+
+delete_image() {
+    local org=$1
+    local repo=$2
+    local tag=$3
+
+    curl --fail-with-body -i -X DELETE \
+        -H "Accept: application/json" \
+        -H "Authorization: JWT $HUB_TOKEN" \
+        "https://hub.docker.com/v2/repositories/$org/$repo/tags/$tag/"
+    
+    echo "Deleted image $org/$repo:$tag"
+}
+
+fetch_tags() {
+    local org=$1
+    local repo=$2
+    local page=$3
+
+    curl -L --fail-with-body \
+        -H "Accept: application/json" \
+        -H "Authorization: JWT $HUB_TOKEN" \
+        "https://hub.docker.com/v2/namespaces/$org/repositories/$repo/tags?page=$page&page_size=100"
+}
+
+page=1
+next_page="non-null"
+
+while [ -n "$next_page" ] && [ "$next_page" != "null" ]; do
```

Comment:
```suggestion
while [ "$next_page" != "null" ]; do
```

We shouldn't need this conditional statement.